### PR TITLE
tweaks

### PR DESCRIPTION
**Pull request by richleland**
* fix/remove links
* update favicon

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,7 +3,12 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="apple-touch-icon" sizes="180x180" href="https://www.sparkpost.com/wp-content/themes/jolteon/favicons/apple-touch-icon.png">
+    <link rel="icon" type="image/png" href="https://www.sparkpost.com/wp-content/themes/jolteon/favicons/favicon-32x32.png" sizes="32x32">
+    <link rel="icon" type="image/png" href="https://www.sparkpost.com/wp-content/themes/jolteon/favicons/favicon-16x16.png" sizes="16x16">
+    <link rel="manifest" href="https://www.sparkpost.com/wp-content/themes/jolteon/favicons/manifest.json">
+    <link rel="mask-icon" href="https://www.sparkpost.com/wp-content/themes/jolteon/favicons/safari-pinned-tab.svg" color="#5bbad5">
+    <link rel="shortcut icon" href="https://www.sparkpost.com/wp-content/themes/jolteon/favicons/favicon.ico">
```
Comment (jasonrhodes) [inline public/index.html]: Should we just grab these and serve them locally?

File: src/components/dkim/ResultDetailHeader.js [hunk 2]
```diff
@@ -26,13 +26,10 @@ class ResultDetailHeader extends Component {
   }
 
   render() {
-    const { loggedIn } = this.props;
```
Comment (jasonrhodes) [inline src/components/dkim/ResultDetailHeader.js]: would it be ok to do something like
```
const { loggedIn, saveEnabled } = this.props; // it'd be false bc undefined

...

{(!loggedIn && saveEnabled) && <ActionLink to='/'>SaveResults</ActionLink>
```

or is that worrying too much and it'll just be simple to add these back when we're ready

Comment (richleland) [inline src/components/dkim/ResultDetailHeader.js]: yeah I'd say just add em back

File: public/index.html (hunk 1 above)
Comment (richleland) [inline public/index.html]: Is there a compelling reason to do it that way? Either way it's easy, but I thought using what was already out there removed some duplication of files.

**Comment by jasonrhodes** [comment]
lgtm, I was gonna say add the redirect query param to the sign up / sign in links but it's so easy to deploy this site I can just add those later whenever.

File: public/index.html (hunk 1 above)
Comment (jasonrhodes) [inline public/index.html]: It's a strange dependency to me, is all -- somebody changes the location of those files, or we rename the theme in WordPress, suddenly tools has broken images. We can decide/update later easily.

**Pull request merged**